site/en/docs/extensions/mv3/tut_debugging/index.md

```diff
@@ -189,7 +189,7 @@ the correct permissions in its [manifest][12].
       "48": "images/get_started48.png",
       "128": "images/get_started128.png"
     },
-    "manifest_version": 2
```

Comment:
This manifest in general is out-of-date: it has `persistent`, and `page_action`, both which should be updated. Since you're here anyway....

Reply:
Other than the screenshots being created from MV2 extensions, I think that is the only issue with them.

Reply:
Yeah, I can update the rest of the out-of-date stuff in the manifest while I'm here!